modules/org/infinispan/dependencies/jdk/openjdk/trim.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-set -e
+#set -e
```

Comment:
We shouldn't uncomment this, as we should handle all error scenarios in the script. If this is removed, then it's a sign the script isn't working as we expect.